$MirOS: ports/archivers/xz/patches/patch-configure_ac,v 1.1 2010/11/20 16:56:06 bsiegert Exp $
--- configure.ac.orig	Fri Jun 22 06:22:06 2012
+++ configure.ac	Thu Oct 18 15:24:57 2012
@@ -15,10 +15,9 @@
 # of malloc(), stat(), or lstat(), since we don't use those functions in
 # a way that would cause the problems the autoconf macros check.
 
-AC_PREREQ([2.64])
-
 AC_INIT([XZ Utils], m4_esyscmd([/bin/sh build-aux/version.sh]),
 	[lasse.collin@tukaani.org], [xz], [http://tukaani.org/xz/])
+AC_DEFINE(PACKAGE_URL, ["http://tukaani.org/xz/"], [The homepage of this package])
 AC_CONFIG_SRCDIR([src/liblzma/common/common.h])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
@@ -416,7 +415,7 @@ fi
 echo
 echo "Initializing Automake:"
 
-AM_INIT_AUTOMAKE([1.10 foreign tar-v7 filename-length-max=99])
+AM_INIT_AUTOMAKE([foreign tar-v7 filename-length-max=99])
 AC_PROG_LN_S
 
 AC_PROG_CC_C99
@@ -443,9 +442,7 @@ fi
 
 echo
 echo "Initializing Libtool:"
-LT_PREREQ([2.2])
-LT_INIT([win32-dll])
-LT_LANG([Windows Resource])
+AC_PROG_LIBTOOL
 
 # This is a bit wrong since it is possible to request that only some libs
 # are built as shared. Using that feature isn't so common though, and this
